utils/bigclown/bigclown-mqtt2influxdb/files/init

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+STOP=1
+
+USE_PROCD=1
+
+PROG=/usr/bin/mqtt2influxdb
+CONF=/etc/bigclown-mqtt2influxdb.yml
+
+start_service() {
+	procd_open_instance
+	procd_set_param respawn ${respawn_threshold:-3600} ${respawn_timeout:-5} ${respawn_retry:-5}
+	procd_set_param command "$PROG" -c "$CONF"
+	procd_set_param stdout 1
+	procd_set_param stderr 1
```

Comment:
just curious here: this means that bigclown is running in the foreground and procd will pump all stderr/stdout to logd;
does bigclown support running into foreground and writting stuff to syslog ?

Reply:
Unfortunately not. Here is a help output as a proof:
```
root@turris-prague:~# bcg --help
Usage: bcg [OPTIONS] COMMAND [ARGS]...

  BigClown gateway between USB serial port and MQTT broker

Options:
  -c, --config FILENAME  configuration file (YAML format).
  -d, --device TEXT      device
  -H, --mqtt-host TEXT   MQTT host to connect to (default is 127.0.0.1)
  -P, --mqtt-port TEXT   MQTT port to connect to (default is 1883)
  --no-wait              no wait on connect or reconnect serial port
  --mqtt-username TEXT   MQTT username
  --mqtt-password TEXT   MQTT password
  --mqtt-cafile TEXT     MQTT cafile
  --mqtt-certfile TEXT   MQTT certfile
  --mqtt-keyfile TEXT    MQTT keyfile
  -v, --verbosity LVL    Either CRITICAL, ERROR, WARNING, INFO or DEBUG
  -D, --debug            Print debug messages, same as --verbosity DEBUG.
  --version              Show the version and exit.
  --help                 Show this message and exit.

Commands:
  devices  Print available devices.
  help     Show help.
```

Reply:
ack
thanks for clarifying :)